vault/migration-bitwarden.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+
+if [[ -z $VAULT_TOKEN ]]; then
+  echo "Please set the varaible $VAULT_TOKEN"
+  exit 1
+fi
+if [[ -z $VAULT_ADDR ]]; then
+  echo "Please set the varaible $VAULT_ADDR"
+  exit 1
+fi
+
+if [[ -z "${@}" ]]; then
+    echo "Usage: migration-bitwarden.sh <bitwarden_path> <type> <vault_path>"
+    echo
+    echo " - bitwarden_path: path of the secret in bitwarden"
+    echo " - type: type of secret (field / pwd)"
+    echo " - vault_path: path of the secret in vault"
+    echo 
+    echo "Ex: migration-bitwarden.sh fleet/waku/test/nodekeys field"
+    echo "Or"
+    echo "migration-bitwarden.sh fleet/waku/test/db/nim-waku pwd"
+    echo 
+    exit 1
+fi
+
+BITWARDEN_PATH="$1"
+TYPE="$2"
+VAULT_PATH="$3"
+VAULT="secret"
+
+extract_field ()
+{
+  while read i; do 
```

Comment:
https://www.shellcheck.net/wiki/SC2162
```suggestion
  while read -r i; do 
```